### DISCUSSION TITLE
Every time I choose to save as MP4 it saves as WebM container

### DISCUSSION THREAD

**Discussion by ryukinix** [deniscerri/ytdlnis#254] [Q&A] [answered]
The applications I try to share the video doesn't support neither webm or mkv containers, I tried to change to other codecs than the default, like fix it as h264, but it didn't work either.

I use the last version v1.6.5 in android 13.

**Comment by deniscerri** [comment, marked as answer]
Try one of the beta in telegram. This has been fixed
